### ISSUE TITLE
MaxListenersExceededWarning: Possible EventEmitter memory leak detected

### ISSUE THREAD

**Issue by XhmikosR** [rollup/rollup#4464]
### Rollup Version

2.70.1

### Operating System (or Browser)

Windows 10 64-bit

### Node Version (if applicable)

16.14.2

### Link To Reproduction

https://github.com/twbs/bootstrap.git

### Expected Behaviour

No `MaxListenersExceededWarning` warning

### Actual Behaviour

Using this script: https://github.com/twbs/bootstrap/blob/main/build/build-plugins.js

```
C:\Users\xmr\Desktop\bootstrap>set NODE_OPTIONS=--trace-warnings && npm run js-compile-plugins

> bootstrap@5.1.3 js-compile-plugins
> node build/build-plugins.js

Building individual plugins...
(node:5572) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 beforeExit listeners added to [process]. Use emitter.setMaxListeners() to increase limit
    at _addListener (node:events:601:17)
    at process.addListener (node:events:623:10)
    at process.once (node:events:667:8)
    at C:\Users\xmr\Desktop\bootstrap\node_modules\rollup\dist\shared\rollup.js:23075:19
    at new Promise (<anonymous>)
    at catchUnfinishedHookActions (C:\Users\xmr\Desktop\bootstrap\node_modules\rollup\dist\shared\rollup.js:23069:35)
    at rollupInternal (C:\Users\xmr\Desktop\bootstrap\node_modules\rollup\dist\shared\rollup.js:23565:11)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async build (C:\Users\xmr\Desktop\bootstrap\build\build-plugins.js:41:18)
    at async Promise.all (index 10)
Built Data
Built Index
Built Manipulator
Built Sanitizer
Built Alert
Built Carousel
Built Button
Built Collapse
Built BaseComponent
Built Dropdown
Built Modal
Built Offcanvas
Built Popover
Built Scrollspy
Built Tab
Built Toast
Built Tooltip
Built EventHandler
Built Backdrop
Built SelectorEngine
Built Config
Built Focustrap
Built ComponentFunctions
Built Scrollbar
Built TemplateFactory
Built Swipe
[build-plugins.js] finished: 1.826s
```

---

Going back to v2.70.0 works without warnings:

```
C:\Users\xmr\Desktop\bootstrap>npm i rollup@2.70.0

changed 1 package, and audited 909 packages in 3s

121 packages are looking for funding
  run `npm fund` for details

6 moderate severity vulnerabilities

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.

C:\Users\xmr\Desktop\bootstrap>npm run js-compile-plugins

> bootstrap@5.1.3 js-compile-plugins
> node build/build-plugins.js

Building individual plugins...
Built Manipulator
Built Index
Built Alert
Built Sanitizer
Built BaseComponent
Built Button
Built Carousel
Built Collapse
Built Dropdown
Built Modal
Built Offcanvas
Built Scrollspy
Built Data
Built Popover
Built Tab
Built Backdrop
Built Toast
Built Tooltip
Built SelectorEngine
Built Config
Built EventHandler
Built Swipe
Built ComponentFunctions
Built Scrollbar
Built TemplateFactory
Built Focustrap
[build-plugins.js] finished: 1.877s
```

**Comment by marijnh** [comment]
Seeing the same warning in CodeMirror's build system after upgrading Rollup. Somewhat annoying that Node does this, but could consider pooling the handlers somehow, or using [`setMaxListeners`](https://nodejs.org/dist/latest-v16.x/docs/api/events.html#emittersetmaxlistenersn) to suppress the warning.

**Comment by XhmikosR** [comment]
It's not a big deal in the sense that it's currently a warning, but it was introduced somewhere here: https://github.com/rollup/rollup/compare/v2.70.0...v2.70.1?w=1

**Comment by lukastaegert** [comment]
Fixed in #4502

**Comment by XhmikosR** [comment]
Thanks @lukastaegert, confirmed!
